### PR TITLE
Added note about multi-region deployment as an option.

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 1]
```diff
@@ -316,7 +316,13 @@ If you have out of scope questions and have not purchased a support contract, yo
 
 If managed FusionAuth hosting does not meet your needs, open a link:/contact/[contact us] with more details.
 
-For instance, if you need longer retention of database backups or static IP addresses capable of being added to a firewall allow/deny list, we're happy to discuss that with you.
+We're happy to discuss custom deployment architectures or configurations. If you need any of these:
+
+* Longer retention of database backups.
+* Static IP addresses capable of being added to a firewall allow/deny list.
+* Multi-region deployments for disaster recovery, compliance or other reasons.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:323]: Multi-region deployments for disaster recovery, compliance, or other reasons.

Comment (jobannon) [reply site/docs/v1/tech/installation-guide/cloud.adoc:323]: comma

File: site/docs/v1/tech/installation-guide/cloud.adoc [hunk 2]
```diff
@@ -316,7 +316,13 @@ If you have out of scope questions and have not purchased a support contract, yo
 
 If managed FusionAuth hosting does not meet your needs, open a link:/contact/[contact us] with more details.
 
-For instance, if you need longer retention of database backups or static IP addresses capable of being added to a firewall allow/deny list, we're happy to discuss that with you.
+We're happy to discuss custom deployment architectures or configurations. If you need any of these:
+
+* Longer retention of database backups.
+* Static IP addresses capable of being added to a firewall allow/deny list.
```
Comment (jobannon) [inline site/docs/v1/tech/installation-guide/cloud.adoc:322]: Do we really support this?

**Review by jobannon — APPROVED** [review]
LGTM!

File: site/docs/v1/tech/installation-guide/cloud.adoc (hunk 2 above)
Comment (mooreds) [reply site/docs/v1/tech/installation-guide/cloud.adoc:322]: We could if someone asked and were willing to pay. Elastic ip address, or run everything through a proxy with a fixed ip address.

**Pull request merged**